console/my-account/reference-content/account-quotas.mdx

```diff
@@ -380,7 +380,7 @@ Transactional Email is a platform that allows Scaleway clients to send [transact
 
 Scaleway’s [Secret Manager](/identity-and-access-management/secret-manager/concepts/#secret-manager/) allows you to conveniently store, access and share sensitive data such as passwords, API keys and certificates.
 
-| Name                                   | [Payment method validated](https://www.scaleway.com/en/docs/console/my-account/how-to/manage-billing-information/#how-to-add-a-credit-card) | Payment method and [identity validated](https://www.scaleway.com/en/docs/console/my-account/how-to/verify-identity/) |
+| Name                                   | [Payment method validated](https://www.scaleway.com/docs/console/my-account/how-to/manage-billing-information/#how-to-add-a-credit-card) | Payment method and [identity validated](https://www.scaleway.com/docs/console/my-account/how-to/verify-identity/) |
```

Comment:
```suggestion
| Name                                   | [Payment method validated](/console/my-account/how-to/manage-billing-information/#how-to-add-a-credit-card) | Payment method and [identity validated](https://www.scaleway.com/docs/console/my-account/how-to/verify-identity/) |
```

Reply:
I think we can remove the `https://www.scaleway.com/docs/` everywhere. No need to specify a full URL